resources/quality/anycubic_4max/petg/anycubic_4max_petg_draft.inst.cfg

```diff
@@ -0,0 +1,20 @@
+[general]
+version = 4
+name = Draft
+definition = anycubic_4max
+
+[metadata]
+setting_version = 5
+type = quality
+quality_type = draft
+weight = 0
+material = generic_petg
+
+[values]
+default_material_print_temperature = 250
```

Comment:
This is not correct, you have to use `material_print_temperature` instead. The `default_material_print_temperature` is defined in the material profiles.